Facet value type (#451)
meilisearch/documentation: guides/advanced_guides/settings.md

  An array of strings that contains the attributes to use as facets.

::: warning

>   Only fields of data type **string** or **array of strings** can be used for faceting.

Reviewer: Formatting alignment?
```suggestion
  Only fields of data type **`String`** or **array of `Strings`** can be used for faceting.
```

Author: I'm not a huge fan. Is this a good practice ?

Reviewer: No I was just wondering. You can resolve it.

>   If the value is `null`, it will be ignored. In any other case, an error will be thrown.

Reviewer: ```suggestion
  A `null` value will be ignored. In any other case, an error will be thrown.
```
